sponsorship_compassion/models/account_invoice.py

```diff
@@ -53,13 +51,31 @@ def _compute_last_payment(self):
     @api.multi
     def _compute_invoice_type(self):
         for invoice in self.filtered(lambda i: i.state in ('open', 'paid')):
-            categories = invoice.with_context(lang='en_US').mapped(
-                'invoice_line_ids.product_id.categ_name')
-            if SPONSORSHIP_CATEGORY in categories:
+
+            sponsorship_cat = self.env.ref('product.product_category_3')
```

Comment:
`product_category_3` is in a demo file and is equivalent to the `Services` category. It is not at all the Sponsorship category.